includes/class-server-rules.php

```
@@ -0,0 +1,117 @@
+			
+			// Reflectively call private get_rules and join it.
+			try {
+				$reflection = new \ReflectionClass( 'PerformanceOptimise\Inc\Htaccess_Handler' );
+				$method = $reflection->getMethod( 'get_rules' );
+				$method->setAccessible( true );
+				$rules = $method->invoke( null );
+				return implode( "\n", $rules );
+			} catch ( \Exception $e ) {
+				return '';
+			}
+		}
```

PHPMD (2.15.0)

[error] The method get_apache_rules is not named in camelCase. (undefined)

(CamelCaseMethodName)